README.md

```diff
@@ -1,2 +1,38 @@
+---
+title: "CFSS hw01"
+author: "Yanning Cui"
+date: "January 9, 2017"
+output: pdf_document
+---
 # hw01
 Repository for first CFSS homework assignment
+
+##This is self introduction of Yanning!
+
+I am a **MAPSS** student and my concentration is *Sociology*.
+
+My research interest is computational content analysis using Twitter API.
+
+To be specific,
+
+- Twitter users' attitudes(**emotions** and **sentiments**) toward a specific event
+- Patterns of attitudes over an amount of time:
+   1.by region(the user's location)
+   2.by gender
+
+##Enough academics!
+> I also love cooking and enjoy music!
+
+Figure 1 is the image of sashimi.
+
+![sashimi](https://s-media-cache-ak0.pinimg.com/736x/c8/c7/27/c8c7270cc815d8967bec0174b4aa7e76.jpg)
+
+You can [link to Wikipedia!](https://en.wikipedia.org/wiki/Sashimi)
+
+##Description
+I followed this [tutorial](https://guides.github.com/features/mastering-markdown/) to use markdown and make essential changes to readme.md.It took me a while to download MacTex (for exporting pdf R markdown files for future use), and restart R, and open a R Markdown file.
+I tried both markdown(.md) and R markdown(.Rmd), and results are the same.I also debug my scripts, and preview pdf/knit pdf to preview the result.
+
+I tried this [tutorial](https://rstudio.github.io/dygraphs/r-markdown.html) to resize the image, but still couldn't get it inserted right after the text in the perfect size. 
```

Comment:
I also attempted to change the size of the image embedded through the markdown document, but was unable to do so. But, when googling, I found out that I could change the image size by using html code.